itou/utils/perms/tests.py

```diff
@@ -0,0 +1,72 @@
+from django.test import TestCase, override_settings
+from django.urls import reverse
+
+from itou.users.factories import UserFactory
+
+
+class UserHijackPermTestCase(TestCase):
+    def test_user_does_not_exist(self):
+        hijacker = UserFactory(is_active=True, is_superuser=True)
+        self.client.force_login(hijacker)
+        response = self.client.post(reverse("hijack:acquire"), {"user_pk": 0, "next": "/foo/"})
+        self.assertEqual(response.status_code, 404)
+
+    def test_superuser(self):
+        hijacked = UserFactory(is_active=True)
+        hijacker = UserFactory(is_active=True, is_superuser=True)
```

Comment:
nit: `is_active=True` est la valeur par défaut, ne pas la spécifier rends les tests plus lisibles.
```suggestion
        hijacked = UserFactory()
        hijacker = UserFactory(is_superuser=True)
```